riscv/insns/cbo_zero.h

```diff
@@ -0,0 +1,11 @@
+require_extension(EXT_ZICBOZ);
+if (((STATE.prv != PRV_M) && (get_field(STATE.menvcfg->read(), MENVCFG_CBZE) == 0)) ||
+    ((STATE.prv == PRV_U && !STATE.v) && (get_field(STATE.senvcfg->read(), SENVCFG_CBZE) == 0)))
+  throw trap_illegal_instruction(insn.bits());
+else if (((STATE.prv == PRV_S && STATE.v) && (get_field(STATE.henvcfg->read(), HENVCFG_CBZE) == 0)) ||
+    ((STATE.prv == PRV_U && STATE.v) && ((get_field(STATE.henvcfg->read(), HENVCFG_CBZE) == 0) ||
+    (get_field(STATE.senvcfg->read(), SENVCFG_CBZE) == 0))))
+  throw trap_virtual_instruction(insn.bits());
+else {
```

Comment:
Shouldn't this one also be replaced with the new macro?

Reply:
Sorry. I forgot to update this.